### PR TITLE
Fix wrong deadline date in check in copy

### PR DESCRIPTION
**Pull request by maxwofford**
Closes https://github.com/hackclub/api/issues/133

### CODE REVIEW

File: app/models/leader.rb [hunk 1]
```diff
@@ -62,6 +62,13 @@ def slack_update
     end
   end
 
+  def timezone
+    Timezone.lookup(latitude, longitude)
+  rescue Timezone::Error::InvalidZone
+    Rails.logger.warn("Unable to find timezone for leader \##{id}")
```
Comment (paked) [inline app/models/leader.rb]: Not sure this log statement is inline with the other precedent in the code base.

File: app/models/hackbot/conversations/check_in.rb [hunk 2]
```diff
@@ -161,6 +160,16 @@ def generate_check_in(event)
 
       private
 
+      def formatted_deadline(lead)
+        timezone = lead.timezone || Timezone.fetch('America/Los_Angeles')
```
Comment (paked) [inline app/models/hackbot/conversations/check_in.rb]: Where does `lead.timezone` come from?

**Review by paked — CHANGES_REQUESTED** [review]

File: app/models/hackbot/conversations/check_in.rb (hunk 2 above)
Comment (MaxWofford) [inline app/models/hackbot/conversations/check_in.rb]: https://github.com/hackclub/api/pull/134/files/bf5ecdd457f157239f4b24bde8bea3cda80873cf#diff-2320146b278b59957e7aae3cc2fd5f84R65

File: app/models/leader.rb (hunk 1 above)
Comment (MaxWofford) [inline app/models/leader.rb]: Let's chat on slack or in-person about this.

File: app/models/hackbot/conversations/check_in.rb (hunk 2 above)
Comment (paked) [inline app/models/hackbot/conversations/check_in.rb]: Oh. Wow. Tripping out apparently.

**Comment by paked** [comment]
@MaxWofford: Can you update this to master?

**Review by paked — APPROVED** [review]

**Comment by MaxWofford** [comment]
Updated!

**Pull request merged**